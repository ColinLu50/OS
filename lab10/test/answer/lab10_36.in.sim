13007
426
294
240
294
238
