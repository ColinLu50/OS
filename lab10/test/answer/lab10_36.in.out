FCFS
54 -> 61 -> 119 -> 24 -> 108 -> 135 -> 112 -> 2 -> 3 -> 211 -> 157 -> 161 -> 66 -> 229 -> 197 -> 7 -> 169 -> 152 -> 129 -> 91 -> 101 -> 116 -> 26 -> 44 -> 207 -> 119 -> 37 -> 203 -> 171 -> 169 -> 166 -> 88 -> 136 -> 154 -> 185 -> 214 -> 202 -> 175 -> 65 -> 205 -> 92 -> 47 -> 42 -> 64 -> 206 -> 57 -> 63 -> 108 -> 240 -> 118 -> 120 -> 113 -> 130 -> 46 -> 192 -> 74 -> 111 -> 79 -> 148 -> 90 -> 214 -> 75 -> 65 -> 164 -> 201 -> 177 -> 42 -> 61 -> 106 -> 216 -> 74 -> 144 -> 185 -> 215 -> 47 -> 33 -> 184 -> 174 -> 19 -> 213 -> 144 -> 219 -> 173 -> 182 -> 235 -> 215 -> 115 -> 125 -> 202 -> 181 -> 20 -> 123 -> 20 -> 85 -> 227 -> 222 -> 226 -> 21 -> 0 -> 223 -> 133 -> 120 -> 97 -> 157 -> 211 -> 61 -> 119 -> 3 -> 177 -> 189 -> 30 -> 123 -> 110 -> 240 -> 184 -> 116 -> 90 -> 16 -> 58 -> 21 -> 80 -> 77 -> 56 -> 188 -> 112 -> 180 -> 146 -> 210 -> 35 -> 118 -> 154 -> 78 -> 75 -> 141 -> 235 -> 3 -> 199 -> 199 -> 76 -> 200 -> 81 -> 88 -> 240 -> 149 -> 86 -> 156 -> 130 -> 225 -> 37 -> 234 -> 191 -> 230 -> 46 -> 223 -> 10 -> 7 -> 58 -> 115 -> 139 -> 141 -> 170 -> 111 -> 189 -> 236 -> 182 -> 77 -> 138 -> 195 -> 72 -> 95 -> 70 -> 232 -> 203 -> 153 -> 211 -> 211
13007
SSTF
54 -> 56 -> 57 -> 58 -> 58 -> 61 -> 61 -> 61 -> 63 -> 64 -> 65 -> 65 -> 66 -> 70 -> 72 -> 74 -> 74 -> 75 -> 75 -> 76 -> 77 -> 77 -> 78 -> 79 -> 80 -> 81 -> 85 -> 86 -> 88 -> 88 -> 90 -> 90 -> 91 -> 92 -> 95 -> 97 -> 101 -> 106 -> 108 -> 108 -> 110 -> 111 -> 111 -> 112 -> 112 -> 113 -> 115 -> 115 -> 116 -> 116 -> 118 -> 118 -> 119 -> 119 -> 119 -> 120 -> 120 -> 123 -> 123 -> 125 -> 129 -> 130 -> 130 -> 133 -> 135 -> 136 -> 138 -> 139 -> 141 -> 141 -> 144 -> 144 -> 146 -> 148 -> 149 -> 152 -> 153 -> 154 -> 154 -> 156 -> 157 -> 157 -> 161 -> 164 -> 166 -> 169 -> 169 -> 170 -> 171 -> 173 -> 174 -> 175 -> 177 -> 177 -> 180 -> 181 -> 182 -> 182 -> 184 -> 184 -> 185 -> 185 -> 188 -> 189 -> 189 -> 191 -> 192 -> 195 -> 197 -> 199 -> 199 -> 200 -> 201 -> 202 -> 202 -> 203 -> 203 -> 205 -> 206 -> 207 -> 210 -> 211 -> 211 -> 211 -> 211 -> 213 -> 214 -> 214 -> 215 -> 215 -> 216 -> 219 -> 222 -> 223 -> 223 -> 225 -> 226 -> 227 -> 229 -> 230 -> 232 -> 234 -> 235 -> 235 -> 236 -> 240 -> 240 -> 240 -> 47 -> 47 -> 46 -> 46 -> 44 -> 42 -> 42 -> 37 -> 37 -> 35 -> 33 -> 30 -> 26 -> 24 -> 21 -> 21 -> 20 -> 20 -> 19 -> 16 -> 10 -> 7 -> 7 -> 3 -> 3 -> 3 -> 2 -> 0
426
SCAN
54 -> 47 -> 46 -> 44 -> 42 -> 37 -> 35 -> 33 -> 30 -> 26 -> 24 -> 21 -> 20 -> 19 -> 16 -> 10 -> 7 -> 3 -> 2 -> 0 -> 0 -> 56 -> 57 -> 58 -> 61 -> 63 -> 64 -> 65 -> 66 -> 70 -> 72 -> 74 -> 75 -> 76 -> 77 -> 78 -> 79 -> 80 -> 81 -> 85 -> 86 -> 88 -> 90 -> 91 -> 92 -> 95 -> 97 -> 101 -> 106 -> 108 -> 110 -> 111 -> 112 -> 113 -> 115 -> 116 -> 118 -> 119 -> 120 -> 123 -> 125 -> 129 -> 130 -> 133 -> 135 -> 136 -> 138 -> 139 -> 141 -> 144 -> 146 -> 148 -> 149 -> 152 -> 153 -> 154 -> 156 -> 157 -> 161 -> 164 -> 166 -> 169 -> 170 -> 171 -> 173 -> 174 -> 175 -> 177 -> 180 -> 181 -> 182 -> 184 -> 185 -> 188 -> 189 -> 191 -> 192 -> 195 -> 197 -> 199 -> 200 -> 201 -> 202 -> 203 -> 205 -> 206 -> 207 -> 210 -> 211 -> 213 -> 214 -> 215 -> 216 -> 219 -> 222 -> 223 -> 225 -> 226 -> 227 -> 229 -> 230 -> 232 -> 234 -> 235 -> 236 -> 240
294
C-SCAN
54 -> 47 -> 46 -> 44 -> 42 -> 37 -> 35 -> 33 -> 30 -> 26 -> 24 -> 21 -> 20 -> 19 -> 16 -> 10 -> 7 -> 3 -> 2 -> 0 -> 0 -> 242 -> 240 -> 236 -> 235 -> 234 -> 232 -> 230 -> 229 -> 227 -> 226 -> 225 -> 223 -> 222 -> 219 -> 216 -> 215 -> 214 -> 213 -> 211 -> 210 -> 207 -> 206 -> 205 -> 203 -> 202 -> 201 -> 200 -> 199 -> 197 -> 195 -> 192 -> 191 -> 189 -> 188 -> 185 -> 184 -> 182 -> 181 -> 180 -> 177 -> 175 -> 174 -> 173 -> 171 -> 170 -> 169 -> 166 -> 164 -> 161 -> 157 -> 156 -> 154 -> 153 -> 152 -> 149 -> 148 -> 146 -> 144 -> 141 -> 139 -> 138 -> 136 -> 135 -> 133 -> 130 -> 129 -> 125 -> 123 -> 120 -> 119 -> 118 -> 116 -> 115 -> 113 -> 112 -> 111 -> 110 -> 108 -> 106 -> 101 -> 97 -> 95 -> 92 -> 91 -> 90 -> 88 -> 86 -> 85 -> 81 -> 80 -> 79 -> 78 -> 77 -> 76 -> 75 -> 74 -> 72 -> 70 -> 66 -> 65 -> 64 -> 63 -> 61 -> 58 -> 57 -> 56
240
LOOK
54 -> 47 -> 46 -> 44 -> 42 -> 37 -> 35 -> 33 -> 30 -> 26 -> 24 -> 21 -> 20 -> 19 -> 16 -> 10 -> 7 -> 3 -> 2 -> 0 -> 56 -> 57 -> 58 -> 61 -> 63 -> 64 -> 65 -> 66 -> 70 -> 72 -> 74 -> 75 -> 76 -> 77 -> 78 -> 79 -> 80 -> 81 -> 85 -> 86 -> 88 -> 90 -> 91 -> 92 -> 95 -> 97 -> 101 -> 106 -> 108 -> 110 -> 111 -> 112 -> 113 -> 115 -> 116 -> 118 -> 119 -> 120 -> 123 -> 125 -> 129 -> 130 -> 133 -> 135 -> 136 -> 138 -> 139 -> 141 -> 144 -> 146 -> 148 -> 149 -> 152 -> 153 -> 154 -> 156 -> 157 -> 161 -> 164 -> 166 -> 169 -> 170 -> 171 -> 173 -> 174 -> 175 -> 177 -> 180 -> 181 -> 182 -> 184 -> 185 -> 188 -> 189 -> 191 -> 192 -> 195 -> 197 -> 199 -> 200 -> 201 -> 202 -> 203 -> 205 -> 206 -> 207 -> 210 -> 211 -> 213 -> 214 -> 215 -> 216 -> 219 -> 222 -> 223 -> 225 -> 226 -> 227 -> 229 -> 230 -> 232 -> 234 -> 235 -> 236 -> 240
294
C-LOOK
54 -> 47 -> 46 -> 44 -> 42 -> 37 -> 35 -> 33 -> 30 -> 26 -> 24 -> 21 -> 20 -> 19 -> 16 -> 10 -> 7 -> 3 -> 2 -> 0 -> 240 -> 236 -> 235 -> 234 -> 232 -> 230 -> 229 -> 227 -> 226 -> 225 -> 223 -> 222 -> 219 -> 216 -> 215 -> 214 -> 213 -> 211 -> 210 -> 207 -> 206 -> 205 -> 203 -> 202 -> 201 -> 200 -> 199 -> 197 -> 195 -> 192 -> 191 -> 189 -> 188 -> 185 -> 184 -> 182 -> 181 -> 180 -> 177 -> 175 -> 174 -> 173 -> 171 -> 170 -> 169 -> 166 -> 164 -> 161 -> 157 -> 156 -> 154 -> 153 -> 152 -> 149 -> 148 -> 146 -> 144 -> 141 -> 139 -> 138 -> 136 -> 135 -> 133 -> 130 -> 129 -> 125 -> 123 -> 120 -> 119 -> 118 -> 116 -> 115 -> 113 -> 112 -> 111 -> 110 -> 108 -> 106 -> 101 -> 97 -> 95 -> 92 -> 91 -> 90 -> 88 -> 86 -> 85 -> 81 -> 80 -> 79 -> 78 -> 77 -> 76 -> 75 -> 74 -> 72 -> 70 -> 66 -> 65 -> 64 -> 63 -> 61 -> 58 -> 57 -> 56
238
